data.tex

```diff
@@ -12,52 +12,63 @@ \section{Early Science Data Products}
-catalog data products are all public and will be available after 24 hours. 
-Rubin Data Rights holders can access both prompt image and catalog data products via the Rubin Science Platform (\S~\ref{ssec:dataaccess}) and query using VO interfaces. 
-Alert are triggered by sources detected with SNR>5 via difference image analysis (DIA) and transmitted to community alert brokers for the list of selected brokers.
+catalog data products are all public and will be available after 24 hours.
+Rubin Data Rights holders can access both prompt image and catalog data products via the Rubin Science Platform (\S~\ref{ssec:dataaccess}) and query using VO interfaces.
+
+Alerts are triggered by sources detected with SNR>5 via difference image analysis (DIA) and transmitted to community alert brokers for the list of selected brokers.
 Alert packets transmitted to these brokers are world-public.
 Similarly, daily Solar System Processing identifies new Solar System Objects from difference image sources and reports those publicly to the Minor Planet Center.
 
+Alert production will begin shortly after System First Light, initially at low latency and limited volume but with both ramping up continuously through commissioning and into the first year of the survey.
+In this way, Alert production and distribution is decoupled from the Data Preview schedule.
+
+In the tables below, we indicate what kind of Alert stream can be expected during each data preview ``era,'' defined as the time period within which that data preview's images are obtained.
+Note that the DP1 era is {\it very short}: the data products released in DP1 will be generated from relatively few observations taken in the few days around System First Light.
+As such, it may not make much sense to run the Prompt Processing pipeline on the DP1 images.
+Instead, the DP2 observations (commissioning's ``system optimization'' and Science Validation surveys) will be able to support prompt processing and alert generation shortly after System First Light.
+DRP image differencing on the First Light image time series will enable some limited initial time domain science with DP1. 
 
 \subsection{Data Release data products}
+
 Static science datasets for Early Science will flow from the \svs in commissioning.
 Images and catalogs from the DRP of the commissioning data will be made available to data rights holders in the form of Data Previews via the access mechanisms described in \S~\ref{ssec:dataaccess}.
 Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to full Data Releases, however all Data Preview data products will be in the same science data model format as for future Data Releases.
 
-
 \subsection{Summary of expected Early Science Data Products} \label{ssec:dataproductsummary}
```

Comment:
We need to remove references to Alerts (canned and live) from these DP tables as we decided to decouple Alert Production from Data previews. The Alerts DPs will be presented in a separate section

Reply:
OK! I can work on that in the DPRPT workbook and generate some new data product tables, in a new PR.